sharkfuser/tests/lit/test_conv_wgrad_asm_emitter_nhwc_krsc_grouped.cpp

```diff
@@ -0,0 +1,135 @@
+// Copyright 2025 Advanced Micro Devices, Inc.
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+// TODO(#2579): This test is disabled due to incorrect lowering of
+//              not unite-stide Grouped ConvWGrad in IREE.
+//              Please enable the test and add LINALG-CHECK section when IREE
+//              supports this case.
+// UNSUPPORTED: {{.*}}
```

Comment:
```suggestion
// XFAIL: {{.*}}
```

This will cause a failure when it passes and should help signal when this test is fixed.